test/abstract_syntax_tree.coffee

```diff
@@ -769,25 +782,6 @@ test "AST as expected for AwaitReturn node", ->
             argument: NUMBER 2
       ]
 
-# test "AST as expected for Value node", ->
```

Comment:
`Value` doesn't need its own section of AST tests, the different types of `Value`'s are tested elsewhere (eg basic values with properties are tested under `Access`)